docs/src/main/paradox/scala/http/server-side/low-level-api.md

```diff
@@ -1,6 +1,6 @@
-# Low-Level Server-Side API
+# Core Server API
```

Comment:
Also good rename, I always thought not many people get the "low / high level" wording we used